easy_rec/python/protos/dataset.proto

```diff
@@ -233,6 +233,10 @@ message DatasetConfig {
     // n data for one feature in tfrecord
     optional uint32 n_data_batch_tfrecord = 24;
 
+    // for csv files, may optionally with an header
+    // in that case, input_name must match header name,
+    // and the number and the order of input_fields
```

Comment:
and the number and the order of input_fields in easy_rec.config

Reply:
input_fields is in this file, they could find it.